CDS/src/org/icpc/tools/cds/CDSConfig.java

```diff
@@ -337,13 +347,18 @@ protected String getUserFromTeamId(String teamId) {
 		if (teamId == null || teamId.isEmpty())
 			return null;
 
-		for (ConfiguredContest cc : contests) {
+		for (IAccount account : userAccounts) {
```

Comment:
The comment above this method is not really correct anymore I think

Reply:
Doh. Fixed javadoc and removed commented out code (an earlier attempt at logging in based on every configured contest...) in new commit.